libraries/AC_WPNav/AC_Circle.h

```diff
@@ -40,17 +40,17 @@ class AC_Circle
     // If conversion fails, defaults to current position and logs a navigation error.
     void set_center(const Location& center);
 
-    // Sets the circle center using a NEU position vector in meters from the EKF origin.
+    // Sets the circle center using a NED position vector in meters from the EKF origin.
     // If `is_terrain_alt` is true, the Z component is treated as altitude above terrain; otherwise, above EKF origin.
```

Comment:
this comment for "is_terrain_alt" also looks out of date.  Perhaps there are many more cases where it's out of date as well.  I'll stop bringing it up :-)

Reply:
looked and found one more.